packages/compass-aggregations/src/modules/aggregation.ts

```diff
@@ -225,6 +225,10 @@ const fetchAggregationData = (page: number): ThunkAction<
         });
       }
     } catch (e) {
+      if ((e as MongoServerError).codeName === 'MaxTimeMSExpired') {
+        track('Aggregation Timed Out');
```

Comment:
Is there any useful metadata to pass here? Maybe the `maxTimeMS` that's set? Fine as is too, mostly raising to see if there's any insights y'all might think of from extra metadata.
If a lot of folks are hitting this event I think that would be useful to see as then maybe we'll want to surface the max time ms setting better (or help people optimize their indexes/pipeline xD).